fe/fe-core/src/main/java/com/starrocks/lake/LakeTablet.java

```diff
@@ -53,11 +53,14 @@ public class LakeTablet extends Tablet {
 
     private static final String JSON_KEY_DATA_SIZE = "dataSize";
     private static final String JSON_KEY_ROW_COUNT = "rowCount";
+    private static final String JSON_KEY_DATA_SIZE_UPDATE_TIME = "dataSizeUpdateTime";
 
     @SerializedName(value = JSON_KEY_DATA_SIZE)
-    private long dataSize = 0L;
+    private volatile long dataSize = 0L;
     @SerializedName(value = JSON_KEY_ROW_COUNT)
-    private long rowCount = 0L;
+    private volatile long rowCount = 0L;
+    @SerializedName(value = JSON_KEY_DATA_SIZE_UPDATE_TIME)
+    private volatile long dataSizeUpdateTime = 0L;
```

Comment:
there will be less metadata size if record this at the partition level.

Reply:
Yes, I agree with you. Since `rowCount` and `dataSize` are saved in `LakeTablet`, here I chose to save `dataSizeUpdateTime` in `LakeTablet` as well.

Reply:
I can think about it some more, in fact I'd like to be able to return these stats in the future via the `PublishVersionResponse`, removing the background collection job here.